tests/test_appctx.py

```diff
@@ -159,6 +159,8 @@ def test_app_ctx_globals_methods(app, app_ctx):
     assert flask.g.pop('bar', 'more cake') == 'more cake'
     # __iter__
     assert list(flask.g) == ['foo']
+    #__repr__
+    assert flask.g.__repr__() == '<flask.g of \'flask_test\'>'
```

Comment:
It should be `repr(flask.g)`.

Reply:
Avoid escaping quotes if not necessary: `"<flask.g of 'flask_test'>"`.

Reply:
If you add a test for when the app context isn't pushed as well, then we'll be covering both branches.